sitemap_index.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <sitemap>
+        <loc>https://www-qa.sap.com/sitemap-articles.xml</loc>
```

Comment:
And here the path must be updated as well.

Reply:
I thought CDN would take care of mapping `https://main--hlx-test--urfuwo.hlx.live/aemedge/sitemap-articles.xml` to `https://www-qa.sap.com/sitemap-articles.xml`. Isn't it so?

Reply:
No, at least this is not planed yet sitemap does not need to be in root. It will be on https://www-qa.sap.com/aemedge/sitemap-...

Only `sitemap_index.xml` is maybe later moved or obsolete as there is already https://www.sap.com/sitemap_index.xml and our sub-sitemaps must be included there. But this we should do as follow up ticket

Reply:
fixed